tools/generator/templates/player/c/api.h.jinja2

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2012-2020 Association Prologin <association@prologin.org>
+{% import 'macros/c.jinja2' as c %}
+
+// This file contains all the API functions for the C language, and all the
+// constants, enumerations and structures.
+// {{ stechec2_generated }}
+
+#pragma once
+
+#include <stdlib.h>
+#include <stdbool.h>
+{% for constant in game.constant %}
+
+{{ c.decl_constant(constant) }}
+{% endfor %}
+{% for enum in game.enum %}
+
+{{ c.decl_enum(enum) }}
+{% endfor %}
+
+{# Forward declaration of structs to build arrays #}
+{% for struct in game.struct %}
+struct {{ struct.str_name }};
+{% endfor %}
+{% for type in array_types(game) %}
+
+typedef struct {{ type }}_array {
+    {%+ if is_struct(type) %}struct {% endif %}{{ type|c_type }}* datas;
```

Comment:
shouldn’t data be singular?

Reply:
This is copied from the old generators but yes. I think "items" is even better.